css/css-content/element-replacement.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<title>The content CSS attribute can replace an element's contents</title>
+<link rel="match" href="element-replacement-ref.html" />
+<link rel="help" href="https://drafts.csswg.org/css-content-3/#content-property" />
+<meta name="assert" content"This test checks that the CSS content propertly can replace a normal element's contents" />
+
+<style>
+p {
+  margin: 0;
+  /* <svg width="100" height="100" version="1.1" xmlns="http://www.w3.org/2000/svg"><rect x="0" y="0" width="100" height="100" fill="green" /></svg> */
+  content: url(data:image/svg+xml,%3Csvg%20width%3D%22100%22%20height%3D%22100%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Crect%20x%3D%220%22%20y%3D%220%22%20width%3D%22100%22%20height%3D%22100%22%20fill%3D%22green%22%20%2F%3E%3C%2Fsvg%3E);
```

Comment:
If the use of a data: URL isn't significant, can you put it in a support directory instead?

Reply:
Done.

Reply:
Forgot to push?

Reply:
Pressed review at the same time as pushing, then the push failed :D. https://github.com/w3c/web-platform-tests/pull/8870/files should have the up-to-date files.